app/helpers/reactjs_helper.rb

```diff
@@ -24,7 +21,25 @@ def webpacked_plugins_with_global_js
   end
 
   def webpacked_plugins_css_for(*plugin_names)
-    css_tags_for(select_requested_plugins(plugin_names)).join.html_safe
+    Foreman::Deprecation.deprecation_warning('3.9', '`webpacked_plugins_css_for` is deprecated, plugin css is already loaded.')
+  end
+
+  def read_webpack_manifest
+    root = File.expand_path(File.dirname(__FILE__) + "/../..")
+    file = File.read(root + '/public/webpack/manifest.json')
+    JSON.parse(file)
```

Comment:
There's `Rails.root` (instance of `Pathname`) as a short hand so it can be simplified to:
```suggestion
    JSON.parse(Rails.root.join('public/webpack/manifest.json').read)
```